### PR TITLE
fix: change header metas and logo

### PR DESCRIPTION
**Pull request by elysee15**
Capture d’écran 2021-09-02 à 17 51 02

### CODE REVIEW

File: web/gds-admin-ui/public/index.html [hunk 1]
```diff
@@ -1,21 +1,18 @@
...
-      name="description"
-      content="Web site created using create-react-app"
-    />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-    <!--
+    <head>
+        <meta charset="utf-8" />
+        <link rel="icon" href="%PUBLIC_URL%/favicon.png" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta name="theme-color" content="#000000" />
```
Comment (bbengfort) [inline web/gds-admin-ui/public/index.html:7]: While we're here, could we please make the theme color #303A46? We might also need to set that in `manifest.json`.

File: web/gds-admin-ui/public/index.html [hunk 2]
```diff
@@ -1,21 +1,18 @@
...
+        <meta name="description" content="Web site created using create-react-app" />
```
Comment (bbengfort) [inline web/gds-admin-ui/public/index.html:8]: For the description:

"Administration for the TRISA Global Directory Service"

File: web/gds-admin-ui/public/index.html [hunk 3]
```diff
@@ -1,21 +1,18 @@
...
+        <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```
Comment (bbengfort) [inline web/gds-admin-ui/public/index.html:9]: Do we need a square version for this image?

File: web/gds-admin-ui/public/index.html [hunk 4]
```diff
@@ -24,12 +21,12 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
-  </head>
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-    <!--
+        <title>React App</title>
```
Comment (bbengfort) [inline web/gds-admin-ui/public/index.html:24]: The title should be "GDS Admin"

File: web/gds-admin-ui/src/assets/scss/app-modern.scss [hunk 5]
```diff
@@ -25,7 +25,7 @@ File: Main Css File
 
 
 //Fonts
-@import "custom/fonts/nunito";
+@import "custom/fonts/posterama";
```
Comment (bbengfort) [inline web/gds-admin-ui/src/assets/scss/app-modern.scss:28]: Very cool that we can use our font!

**Review by bbengfort — COMMENTED** [review]
Since we're here - would you also mind ensuring that robots.txt is

```
User-agent: *
Disallow: /
```

Thanks for these updates - looking good!

File: web/gds-admin-ui/public/index.html (hunk 3 above)
Comment (elysee15) [reply web/gds-admin-ui/public/index.html:9]: No

**Review by bbengfort — APPROVED** [review]
Thank you for making those changes!

**Pull request merged**